webapp/loggedin/surveyViews/categoryEdit/components/categoryEditView.scss

```diff
@@ -83,13 +83,18 @@ $borderColor: $greyBorder;
 
   .btn-close {
     position: absolute;
-    right: 0px;
-    top: 0px;
+    right: 0;
+    top: 0;
   }
 
   .btn-delete {
     justify-self: center;
   }
 
+  > div {
```

Comment:
why not to add the ```.ellipsis``` class to the div instead of adding the same rule here?

Reply:
good to know it exists...!